### DISCUSSION TITLE
WebSocket

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#1060] [Bug Submission]
### Describe the bug

Notifications can't be used, is the error from me or from you, I couldn't find a solution

### Reproduction

![IMG_20220126_184439](https://user-images.githubusercontent.com/97207588/151197391-4071f084-065d-480c-af79-d02e20ef361c.jpg)

### System Info

```shell
Notifications are unavailable
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by powerpook** [comment]
Maybe your network is unstable. Check your network
![png1](https://user-images.githubusercontent.com/75034263/151279057-36458ac1-df2b-4462-a80f-08c76f747b23.png)
![png2](https://user-images.githubusercontent.com/75034263/151279090-3d3181b2-775a-4281-a65e-ddb4faa5b2aa.png)

**Comment by ibaxio** [comment]
Please try again or change to another Test Network.
You can find the Honer Node list and port on http://scan.ibax.io
